translations/ui-orders/en_US.json

```diff
@@ -644,5 +644,7 @@
     "eresource.url": "URL",
     "renewals.subscription.tooltip": "Choose Order type as Ongoing to interact with this field",
     "renewals.manualRenewal.tooltip": "Check Subscription to interact with this field",
-    "orderSummary.totalExpended": "Total expended"
+    "orderSummary.totalExpended": "Total expended",
+    "orderLineList.orderWorkflow": "Order workflow",
```

Comment:
it looks like wrong merge, it should be "orderLineList.orderWorkflow": "Order status",

Reply:
At some point it was "Order workflow" and then was changed: https://github.com/folio-org/ui-orders/commit/6992b8505dad803eadca848d6d1a267b74686473 — I wonder if the change happened as I was pulling translations.  I will run another batch update from Lokalise.  There shouldn't be too many changes from over the weekend.